src/main/java/com/mastery/java/task/Simplewebapp.java

```diff
@@ -0,0 +1,22 @@
+package com.mastery.java.task;
+
+import org.apache.activemq.command.ActiveMQQueue;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.jms.annotation.EnableJms;
+
+import javax.jms.Queue;
+
+@SpringBootApplication
+@EnableJms
+public class Simplewebapp {
```

Comment:
please do format all the application code to add readability

Reply:
Done, Ctrl + Alt + L is miracle)